frontend/src/metabase/public/components/widgets/EmbedModalContent.jsx

```diff
@@ -235,7 +235,20 @@ class EmbedModalContent extends Component {
   }
 }
 
-export default connect(mapStateToProps)(EmbedModalContent);
+function getDefaultEmbeddingParams(props) {
+  const { resource, resourceParameters } = props;
+
+  return filterValidResourceParameters(
+    resource.embedding_params || {},
+    resourceParameters,
+  );
+}
+
+function filterValidResourceParameters(embeddingParams, resourceParameters) {
+  const validParameters = resourceParameters.map(parameter => parameter.slug);
```

Comment:
the value that's set in `embedding_params` is from `parameter.slug`

https://github.com/metabase/metabase/blob/8c7d0bcdd6526201d5ab2349a42f8305fbfd52a8/frontend/src/metabase/public/components/widgets/AdvancedSettingsPane.jsx#L81